src/schedulers.jl

```diff
@@ -56,10 +57,10 @@ with other multithreaded code.
 - `chunksize::Integer` (default not set)
     * Specifies the desired chunk size (instead of the number of chunks).
     * The options `chunksize` and `nchunks`/`ntasks` are **mutually exclusive** (only one may be a positive integer).
-- `split::Symbol` (default `:batch`):
+- `split::OhMyThreads.Split` (default `OhMyThreads.Consecutive()`):
```

Comment:
might be worth mentioning that we support a `Symbol` here
```suggestion
- `split::Union{Symbol, OhMyThreads.Split}` (default `OhMyThreads.Consecutive()`):
```